cmd/kubeadm/app/util/error.go

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright 2014 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+	"os"
+	"os/user"
+	"strings"
+
+	"github.com/golang/glog"
+	"github.com/renstrom/dedent"
+)
+
+const (
+	DefaultErrorExitCode = 1
+	EX_USAGE             = 64
+	EX_DATAERR           = 65
+	EX_NOINPUT           = 66
+	EX_NOUSER            = 67
+	EX_NOHOST            = 68
+	EX_UNAVAILABLE       = 69
+	EX_SOFTWARE          = 70
+	EX_OSERR             = 71
+	EX_OSFILE            = 72
+	EX_CANTCREAT         = 73
+	EX_IOERR             = 74
+	EX_TEMPFAIL          = 75
+	EX_PROTOCOL          = 76
+	EX_NOPERM            = 77
+	EX_CONFIG            = 78
+)
+
+var AlphaWarningOnExit = dedent.Dedent(`
+	kubeadm: I am an alpha version, my authors welcome your feedback and bug reports
+	kubeadm: please create issue an using https://github.com/kubernetes/kubernetes/issues/new
+	kubeadm: and make sure to mention @kubernetes/sig-cluster-lifecycle. Thank you!
+`)
+
+type NotRootError struct {
```

Comment:
Do you plan adding structs for every possible error message?
Seems like that's going to be a maintenance heavy thing to do.

I think I'm in favor for plain Go-style error handling